erpnext/stock/doctype/item/item.py

```diff
@@ -898,6 +905,72 @@ def validate_auto_reorder_enabled_in_stock_settings(self):
 	def validate_valuation_method(self):
 		if self.is_new() and not self.valuation_method:
 			self.valuation_method = frappe.db.get_single_value("Stock Settings", "valuation_method")
+	
+	def update_to_bloomtrace(self):
+		"""
+		Create a new item on Bloomtrace via Bloomstack.
+		Args: 
+		
+		Returns: 
+			Successful Message to the User
+		"""
+		try:
+			data = self.as_dict()
+			request_data = {
+				"site_url": "manufacturing.bloomstack.io",
+				"customer_name": "Bloomstack",
+				"company_name": "Bloomstack India",
+				"license_number": "A12-0000015-LIC",
+				"rest_method": "POST",
+				"environment": "sandbox",
+				"doctype": self.doctype,
+				"doctype_data": data
+			}
+
+			# to map the metrc_item_name, metrc_uom, metrc_item_category
+			request_data['doctype_data']['name'] = self.metrc_item_name
+			request_data['doctype_data']['uom'] = self.metrc_uom
+			request_data['doctype_data']['item_category'] = self.metrc_item_category
+
+			# passing the hard coded modified_by
+			request_data['doctype_data']['modified_by'] = "neil@bloomstack.com"
```

Comment:
this cannot be hard coded

Reply:
only this account can get the user key from admin portal or else it's failing at metrc post, also we cannot put the owner/modified by in the bloomtrace settings, right??